backend/cfg/trap_stack.ml

```diff
@@ -192,20 +192,23 @@ module Make (D : D) = struct
 
   let link ~(src : t) ~(dst : t) =
     assert (!src = Unknown);
```

Comment:
From the pull request description, I would expect
the code to still fail if this `assert` is not moved to
the `else` part of the function.

Reply:
The assert only checks that `src` is unknown. It's only a problem when both `src` and `dst` are unknown.